sql/moz-fx-data-shared-prod/operational_monitoring_derived/dataset_metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: Operational Monitoring Derived
+description: |-
+  Derived data based operational monitoring project requirements
```

Comment:
It might be useful to add a mention here that the queries are generated via an Airflow task and link to it. Same for the operational_monitoring dataset